bin/run.sh

```diff
@@ -17,5 +17,32 @@ else
   fi
 fi
 
+
+if [[ -f VERSION ]]
+then
+  VERSION=$(cat VERSION)
+else
+  VERSION="Manual Deployment"
+fi
+
+NEW_RELIC_API_KEY=$(
+  echo "${VCAP_SERVICES}" | \
+  jq '.[] | map(select(.name == "tock-credentials-logging")) | .[].credentials.NEW_RELIC_API_KEY' -r
+)
+
+# Append New Relic API key to New Relic INI file for `newrelic-admin` tool
+cat <<EOF >> "${NEW_RELIC_CONFIG_FILE}"
+
+# Adding license key from script $(basename "${0}")
+api_key=${NEW_RELIC_API_KEY}
+EOF
```

Comment:
Is this additional New Relic setup needed?  Or is the initialization in [`wsgi.py`](https://github.com/18F/tock/blob/master/tock/tock/wsgi.py#L16-L25) enough?

Reply:
This is specific for the command on line 45, [see the docs here](https://docs.newrelic.com/docs/agents/python-agent/installation-configuration/python-agent-admin-script#record-deploy).